### PR TITLE
Fix:589 Transfer the options when invoking hints

### PR DESCRIPTION
**Pull request by sirfilip**
#589 proposed fix

### CODE REVIEW

File: spec/extensions/hints/result_spec.rb [hunk 1]
```diff
@@ -52,6 +53,31 @@
...
+                                    dry_validation:
+                                      errors:
+                                        filled?: "de - must be filled"
+                                        size?: "de - length must be within 2 - 4"
+                                  HEREDOC
+                                }
+
+        it 'returns the correct translations' do
+          allow(YAML).to receive(:load_file).and_return(YAML.load(StringIO.new(yaml_messages)))
+          expect(result.errors(locale: :de).to_h).to eq(name: ['de - must be filled'])
```
Comment (solnic) [inline spec/extensions/hints/result_spec.rb:75]:
```suggestion
          expect(result.errors(locale: :de).to_h).to eql(name: ['de - must be filled'])
```

File: spec/extensions/hints/result_spec.rb [hunk 2]
```diff
@@ -52,6 +53,31 @@
...
+          expect(result.messages(locale: :de).to_h).to eql(name: ["de - must be filled", "de - length must be within 2 - 4"])
+          expect(result.errors(locale: :en).to_h).to eq(name: ['must be filled'])
```
Comment (solnic) [inline spec/extensions/hints/result_spec.rb:77]:
```suggestion
          expect(result.errors(locale: :en).to_h).to eql(name: ['must be filled'])
```

File: spec/extensions/hints/result_spec.rb [hunk 3]
```diff
@@ -52,6 +53,31 @@
...
+                                        filled?: "must be filled"
+                                        size?: "length must be within 2 - 4"
+
+                                  de:
+                                    dry_validation:
+                                      errors:
+                                        filled?: "de - must be filled"
+                                        size?: "de - length must be within 2 - 4"
+                                  HEREDOC
+                                }
```
Comment (solnic) [inline spec/extensions/hints/result_spec.rb:71]: Could you reformat this so that it's a do...end block and the yaml uses less indentation?

Comment (sirfilip) [reply spec/extensions/hints/result_spec.rb:71]: Yup i am on it

**Comment by solnic** [comment]
Don't worry about failing CI. This is a problem with configuration that I need to fix in master.

**Comment by solnic** [comment]
Please rebase and squash the commits - this should fix the CI and keep the history clean.

**Pull request merged**